motioneye/static/js/main.js

```diff
@@ -5084,17 +5088,47 @@ function addCameraFrameUi(cameraConfig) {
         }
     };
 
+    /* check if action buttons need one or two rows*/
+    if(cameraConfig.actions.length) {
+        var actionButtonWidth = actionButtonDict[cameraConfig.actions[0]].height();
+        var actionsWidth = actionsCount * actionButtonWidth;
+        var cameraWidth = cameraOverlay.find('div.camera-overlay-bottom').width();
+        var flexBasis = (200 / actionsCount);
+        // to account for uneven numbers to space correctly
+        if(actionsCount % 2 == 1){
+            switch (Math.floor(actionsCount/2)) {
+                case 4:
+                    flexBasis -= 4;
+                    break;
+                case 5:
+                    flexBasis -= 2.5;
+                    break;
+                case 6:
+                case 7:
+                    flexBasis -= 1.5;
+                    break;
+            }
+        }
+        flexBasis += '%';
+        if(cameraWidth <= actionsWidth){
+            actionButtonDiv.css('flex', '1 0' + flexBasis);
+        } else {
+            actionButtonDiv.css('flex', 'none');
+        }
+    }
+
     cameraImg.addClass('initializing');
     cameraImg[0].initializing = true;
     cameraImg.height(Math.round(cameraImg.width() * 0.75));
 }
 
-function remCameraFrameUi(cameraId) {
-    var cameraFrameDiv = getPageContainer().find('div.camera-frame#camera' + cameraId);
-    cameraFrameDiv.animate({'opacity': 0}, 100, function () {
-        cameraFrameDiv.remove();
-    });
-}
+// was removed 10 years ago, needs cleanup, a862509fa816049d1fa6d7da1712b2a6c5e34519
+// function remCameraFrameUi(cameraId) {
+//     var cameraFrameDiv = getPageContainer().find('div.camera-frame#camera' + cameraId);
+//     cameraFrameDiv.animate({'opacity': 0}, 100, function () {
+//         cameraFrameDiv.remove();
+//     });
+// }
 
 function recreateCameraFrames(cameras) {
```

Comment:
```suggestion
function recreateCameraFrames(cameras) {
```

Reply:
Ok this is really not used anymore. I removed it 👍